app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -195,7 +195,7 @@ def parse_resource_group(resource_group)
       end
 
       def parse_series(s)
-        name = uid = s.name
+        name = uid = s.name.downcase
```

Comment:
This might cause unintentional thrashing on upgrade.  That is, all of the old series will be deleted, and then they will all be readded (because the ems_refs won't match).  You may need a data migration to handle this.